include/swift/AST/CMakeLists.txt

```diff
@@ -5,6 +5,5 @@ else()
 endif()
 
 set(generated_include_sources
-    Attr.def.gyb
     TokenKinds.def.gyb)
```

Comment:
`TokenKinds.def.gyb` doesn’t exist anymore either. I removed both of them in https://github.com/apple/swift/pull/67666

Reply:
Perfect, I'll remove that as well!

Reply:
✅